python/sglang/srt/server.py

```diff
@@ -459,6 +459,7 @@ def launch_server(
         add_api_key_middleware(app, server_args.api_key)
 
     # add prometheus middleware
+    _set_prometheus_env()
     add_prometheus_middleware(app)
```

Comment:
should we only enable this when `enable_metrics`?

Reply:
yes